requirements/docs.txt

```diff
@@ -1,12 +1,12 @@
 # These are dependencies of various sphinx extensions for documentation.
-sphinx>=4.3
+sphinx==5.0.0
 ipython
 matplotlib
 sphinx-copybutton
-sphinx-gallery>=0.10.0
-pydata-sphinx-theme>=0.10.1
+sphinx-gallery==0.11.0
+pydata-sphinx-theme==0.10.1
```

Comment:
I am working with `pydata-sphinx-theme==0.12.0` and all is good.  However, the glitch comes often from here